### ISSUE TITLE
Background disapear after HA reboot using severity

### ISSUE THREAD

**Issue by lamirand-g** [custom-cards/bar-card#5]
When using a severity settings, it seems the order of value/color is important. However after HA reboot is always sort naturaly and so the bar-card is broken. If aI change back the order everything works as expected.

![image](https://user-images.githubusercontent.com/4324887/55782753-4251a600-5aad-11e9-944c-4a340e38e885.png)

```
- align: split
    entity: sensor.temperature_sda
    icon: 'mdi:thermometer'
    severity:
      - color: '#40bf40'
        value: 50
      - color: var(--label-badge-yellow)
        value: 60
      - color: '#bf4040'  
        value: 100              
    type: 'custom:bar-card'
```

**Comment by lamirand-g** [comment]
I have the following:

```html
<div id="bar_sensor_temperature_sda_0" style="--bar-percent:33%; --bar-charge-percent:33%; --bar-color:hsl(undefined,50%,50%);"></div>
```

**Comment by Gluwc** [comment]
👍

**Comment by Gluwc** [comment]
Ok that's strange. I'm going to need some more help from you to debug this since I've never seen this behaviour.
Could you try [this](https://gist.github.com/Gluwc/6279200c08991755e3cf6d587dcbf0ff) gist, I've added some console log calls to the parts the return the color to try and figure out what's going on. Could you check the debug console and let me know what it reports?

**Comment by lamirand-g** [comment]
No problem.

This is what I got from the debug console. (note that I have many bar-card, that I guess the reason there are many output)

![image](https://user-images.githubusercontent.com/4324887/55820353-3ee50b80-5afb-11e9-97c9-3268de55fc3f.png)

**Comment by Gluwc** [comment]
Right so that looks just fine. Are your bars working with this config?

Also can you confirm if you're using the GUI or yaml mode for lovelace. Since you're saying the it's re-arranging the severity order, my guess is GUI?

This is a side-effect of the way the GUI mode stores the config in JSON format which is probably sorts alphabetically on start-up. I will add some code in an upcoming release to sort it from low to high in the card itself so this shouldn't be an issue anymore.

**Comment by Gluwc** [comment]
Actually I think I know what's happening now. My card is checking the 2nd value of every severity entry, which in your case is sorted to the 1st value. This should be a fairly easy fix.

**Comment by Gluwc** [comment]
Whoops closed by accident. I will keep this open until I've submitted a fix.

**Comment by Gluwc** [comment]
Should be fixed in [0.2.0b0](https://github.com/custom-cards/bar-card/releases/tag/0.2.0b0). Please confirm and I'll close the issue.

**Comment by lamirand-g** [comment]
Sorry for the delay but I confirm that the issue is now fixed. 
Thanks
